src/components/home/LevelCollectionsPage.js

```diff
@@ -5,31 +5,23 @@ import { useTranslation } from 'react-i18next';
 import CollectionsGrid from '../collection/CollectionsGrid';
 import { QueryClientContext } from '../QueryClientContext';
 
-const LevelCollectionsPage = ({ selectedOptions }) => {
+const LevelCollectionsPage = ({ selectedLevel, selectedDiscipline }) => {
   const { t } = useTranslation();
   const { hooks } = useContext(QueryClientContext);
 
-  // get all categories
-  const { data: allCategories } = hooks.useCategories();
-
-  // get category map (to map between category id and name)
-  const categoriesMap = new Map(
-    allCategories?.map((entry) => [entry.name, entry.id]),
-  );
-
-  const categories = selectedOptions?.filter(Boolean);
-  const categoryIds = categories?.map((entry) => categoriesMap?.get(entry));
+  // join all selected categories by comma
+  const categoryIds = [
+    selectedDiscipline.join(','),
+    selectedLevel.join(','),
+  ].filter(Boolean);
```

Comment:
It's not good to rely on index to provide ids by category types. 
Can't you use a single string instead?

Reply:
The category types doesn't matter here. It's just group the ids belong to the same type together, the order is arbitrary